src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import ball.characteristics.Ball;
+import ball.characteristics.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        Ball ball = new Ball();
```

Comment:
It's unnecessary to create a new instance of `Ball` in this method. You can directly return the color as a string using `Color.values()[index].name()`.

Reply:
good advice